modules/unsupported/geojsonstore/src/main/java/com/bedatadriven/jackson/datatype/jts/GeoJson.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+/*
```

Comment:
Hum... is it sufficient to link the original source code or do we have to have both license headers?
But I see, the original code did not have license headers either: 
https://github.com/bedatadriven/jackson-datatype-jts/blob/master/src/main/java/com/bedatadriven/jackson/datatype/jts/GeoJson.java

Thinking out loud here: do we need to preserve the original package names?

Reply:
@jodygarnett recommended this in the previous PR. As for the package names I didn't think of changing them but there is no reason not to I guess.